prow/cmd/peribolos/OWNERS

```diff
@@ -4,5 +4,7 @@ reviewers:
 - cjwagner
 approvers:
 - cjwagner
+- MadhavJivrajani
+- Priyankasaggu11929
```

Comment:
We can replace this with an alias: https://github.com/kubernetes/test-infra/blob/9fd28d825479145276bcd5f64b16b0c49231971d/OWNERS_ALIASES#L68C3-L68C46

Reply:
@Priyankasaggu11929 I like Madhav's suggestion. Let's use this alias.

Reply:
updated. Thanks! :pray: